params/version.go

```diff
@@ -20,32 +20,32 @@ import (
 	"fmt"
 )
 
-const (
+var (
 	VersionMajor = 1        // Major version component of the current release
 	VersionMinor = 14       // Minor version component of the current release
 	VersionPatch = 3        // Patch version component of the current release
 	VersionMeta  = "stable" // Version metadata to append to the version string
 )
 
 // Version holds the textual version string.
-var Version = func() string {
+func Version() string {
 	return fmt.Sprintf("%d.%d.%d", VersionMajor, VersionMinor, VersionPatch)
-}()
+}
 
 // VersionWithMeta holds the textual version string including the metadata.
-var VersionWithMeta = func() string {
-	v := Version
+func VersionWithMeta() string {
+	v := Version()
 	if VersionMeta != "" {
 		v += "-" + VersionMeta
 	}
 	return v
-}()
+}
```

Comment:
Are these changes needed?

Reply:
Reworked to not need version mutability in ethereum at all...